pkg/services/control/types.proto

```diff
@@ -141,6 +141,10 @@ message ShardInfo {
     string pilorama_path = 7 [json_name = "piloramaPath"];
 }
 
+message ObjectInfo {
+    string address = 1 [json_name = "object address"];
```

Comment:
dont mind `string` in general but a raw byte slice can be a little bit more efficient here